portfolio/src/main/webapp/about.js

```diff
@@ -0,0 +1,82 @@
+function loadInFact(index, fact_index, isTruth) {
+    let fact = isTruth ? truths[fact_index] : lies[fact_index];
+
+    switch(index) {
+        case 0:
+            document.getElementById("first_txt").innerHTML = fact[0];
```

Comment:
Here, and in the rest of this file (and all JS files), the style guide calls for using single quotes in javascript.  (double quotes are used in HTML files)

Haha... seemingly arbitrary?  But that's what the great Google above has decided...

https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

Reply:
Changed!